distro/docker-compose/.env.template

```diff
@@ -39,7 +39,7 @@ APICURIO_UI_KC_CLIENT_ID=apicurio-studio
 APICURIO_UI_HUB_API_URL=http://localhost:8091
 APICURIO_UI_EDITING_URL=ws://localhost:8092
 APICURIO_UI_FEATURE_MICROCKS=true
-APICURIO_UI_VALIDATION_CHANNELNAME_REGEXP='([^\x00-\x20\x7f"'\''%<>\\^`{|}]|%[0-9A-Fa-f]{2}|\{[+#./;?&=,!@|]?((\w|%[0-9A-Fa-f]{2})(\.?(\w|%[0-9A-Fa-f]{2}))*(:[1-9]\d{0,3}|\*)?)(,((\w|%[0-9A-Fa-f]{2})(\.?(\w|%[0-9A-Fa-f]{2}))*(:[1-9]\d{0,3}|\*)?))*\})*'
```

Comment:
Side note:  we've gotta get rid of this or simplify it somehow.  It's caused a number of issues...

Reply:
Hi Eric, thanks for reviewing. Are you referring to getting rid of the approach I used to modify the regular expression, or the original value itself that needs to be simplified?